FDA/FDA/ContentView.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        viewstate.PlacesView()
```

Comment:
Odkial beries ten `viewState` ? to je nejaka global promenna? To je nejaka kulišárna 🤔

Reply:
a kde injectujes tie `EnvironmentObject` ?